### PR TITLE
Added navbar and modal component

### PR DESCRIPTION
**Pull request by kaustubh-jsr**

### CODE REVIEW

File: components/modal.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+@import "https://mantra-ui.netlify.app/style.css";
+
+#show-modal-btn {
+  width: max-content;
+}
```
Comment (yaswanthmyneni) [inline components/modal.css:5]: changes: avoid giving styles to id's.

File: documentation.html [hunk 2]
```diff
@@ -648,6 +650,111 @@ <h1 class="card--heading">Strawberries</h1>
...
+                          </li>
+                          <li>
+                            <a href="#" class="fab fa-github"
+                              ></a
+                            >
+                          </li>
+                        </ul>
+                      </nav>
+                </div>
+                <div class="component__example-code">
```
Comment (yaswanthmyneni) [inline documentation.html:704]: suggestion: don't use this kind of naming for classes, use kebab casing.

**Review by yaswanthmyneni — APPROVED** [review]
you can merge after changes.

File: components/modal.css (hunk 1 above)
Comment (kaustubh-jsr) [reply components/modal.css:5]: Done, changed from id to class.

File: documentation.html (hunk 2 above)
Comment (kaustubh-jsr) [reply documentation.html:704]: In this library I have used similar naming throughout the project, but will keep this thing in mind. Thanks for the suggestion.

**Review by omkarpatke — APPROVED** [review]
I can't see any mistakes so do the changes that Yashwant suggested and feel free to merge.

**Review by rdthetha — COMMENTED** [review]
image
The navbar is overlapping with the modal in the preview.

File: components/modal.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  opacity: 0;
+  pointer-events: none;
+  backdrop-filter: blur(1px);
+}
+
+.modal-container.show {
+  opacity: 1;
+  pointer-events: auto;
+}
+
```
Comment (rdthetha) [inline components/modal.css]: Can give higher z-index to the modal container!

Comment (kaustubh-jsr) [reply components/modal.css]: Thanks for pointing this out. Resolved it.

**Pull request merged**